lib/internal/errors.js

```diff
@@ -162,6 +225,7 @@ E('ERR_SOCKET_BAD_TYPE',
 E('ERR_SOCKET_CANNOT_SEND', 'Unable to send data');
 E('ERR_SOCKET_CLOSED', 'Socket is closed');
 E('ERR_SOCKET_DGRAM_NOT_RUNNING', 'Not running');
+E('ERR_OUTOFMEMORY', 'Out of memory');
```

Comment:
They are... it's in my todo list to make sure these all get sorted before this lands.